src/garage/torch/modules/tanh_gaussian_mlp_module.py

```diff
@@ -0,0 +1,242 @@
+            in the std network.
+        std_hidden_w_init (callable):  Initializer function for the weight
+            of hidden layer (s).
+        std_hidden_b_init (callable): Initializer function for the bias
+            of intermediate dense layer(s).
+        std_output_w_init (callable): Initializer function for the weight
+            of output dense layer(s) in the std network.
+        std_parameterization (str): How the std should be parametrized. There
+            are two options:
+            - exp: the logarithm of the std will be stored, and applied a
+               exponential transformation
+            - softplus: the std will be computed as log(1+exp(x))
+        layer_normalization (bool): Bool for using layer normalization or not.
+
+    """
+
+    def __init__(self,
+                 input_dim,
+                 output_dim,
+                 hidden_sizes=(32, 32),
+                 hidden_nonlinearity=torch.tanh,
+                 hidden_w_init=nn.init.xavier_uniform_,
+                 hidden_b_init=nn.init.zeros_,
+                 output_nonlinearity=None,
+                 output_w_init=nn.init.xavier_uniform_,
+                 output_b_init=nn.init.zeros_,
+                 init_std=1.0,
+                 min_std=1e-6,
+                 max_std=None,
+                 learn_std=True,
+                 std_hidden_sizes=(32, 32),
+                 std_hidden_nonlinearity=nn.ReLU,
+                 std_hidden_w_init=nn.init.xavier_uniform_,
+                 std_hidden_b_init=nn.init.zeros_,
+                 std_output_w_init=nn.init.xavier_uniform_,
+                 std_parameterization='exp',
+                 layer_normalization=False):
+        super().__init__(input_dim=input_dim,
+                         output_dim=output_dim,
+                         hidden_sizes=hidden_sizes,
+                         hidden_nonlinearity=hidden_nonlinearity,
+                         hidden_w_init=hidden_w_init,
+                         hidden_b_init=hidden_b_init,
+                         output_nonlinearity=output_nonlinearity,
+                         output_w_init=output_w_init,
+                         output_b_init=output_b_init,
+                         learn_std=learn_std,
+                         init_std=init_std,
+                         min_std=max_std,
+                         max_std=max_std,
+                         std_hidden_sizes=std_hidden_sizes,
+                         std_hidden_nonlinearity=std_hidden_nonlinearity,
+                         std_hidden_w_init=std_hidden_w_init,
+                         std_hidden_b_init=std_hidden_b_init,
+                         std_output_nonlinearity=torch.tanh,
+                         std_output_w_init=std_output_w_init,
+                         std_parameterization=std_parameterization,
+                         layer_normalization=layer_normalization)
+
+    def forward(self, *inputs):
```

Comment:
why would you take *inputs and then only use inputs[0]? just name the single input you're going to take.